book-2-levelup/chapters/LU_AUTHENTICATION.md

```diff
@@ -97,11 +102,14 @@ def register_user(request):
 
 ## Auth Package Imports
 
-The `auth` directory is going to become a package so that other modules can import the functions. Create the `views/__init__.py` file and place the following code in it.
+The `auth` directory is going to become a package so that other modules can import the functions.
```

Comment:
```suggestion
The `views` directory is going to become a package so that other modules can import the functions. We tell our code it should be a module by adding an `__init__.py` file to the directory.
```